packages/taquito-beacon-wallet/src/taquito-beacon-wallet.ts

```diff
@@ -182,4 +210,42 @@ export class BeaconWallet implements WalletProvider {
   async clearActiveAccount() {
     await this.client.setActiveAccount();
   }
+
+  async sign(signingRequest: { payload: string }) {
```

Comment:
Understand  the intention it's good consistency for signer and wallet alike's sign function to return the same object format. Also feel like for people having a front end app with connect wallet (use of wallet api) to sign something they might only make use prefix signature which is returned from `this.client.requestSignPayload(..)`. Can be down to preference, just would like to voice out

Reply:
What do you suggest? To not do all the calculations, and only return the `prefixSignature`? Then if we want to send it to the node, we need to compute `sbytes` anyways.